pages/common/wormhole.md

```diff
@@ -0,0 +1,23 @@
+# wormhole
+
+> Share files from one computer to another, safely.
```

Comment:
How about "Share files safely from one compute to another."

Reply:
*compute -> computer?

Reply:
woops, yeah.